src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Redirect, Switch } from 'react-router-dom';
+import LoginPage from './LoginPage';
+import DashboardPage from './DashboardPage';
+
+function App() {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/login" component={LoginPage} />
```

Comment:
Consider adding a 404 Not Found route for unmatched paths for better user experience.